Polyfills/Canvas/Source/Canvas.cpp

```diff
@@ -59,22 +59,23 @@ namespace Babylon::Polyfills::Internal
         // called when removed from document which has no meaning for Native
     }
 
+    // this is currently synchronous to work around font corruption issues
```

Comment:
Probably don't even need this comment as it is totally fine for it to be synchronous with the current impl. The previous asynchrony did not server any purpose.